memorystore/redis/README.md

```diff
@@ -13,3 +13,21 @@ Follow the instructions in [this guide](https://cloud.google.com/memorystore/doc
 ## Running on Google App Engine Flex
 
 Follow the instructions in [this guide](https://cloud.google.com/memorystore/docs/redis/connect-redis-instance-flex) to deploy the sample application on GAE Flex.
+
+## Running Locally
+
+To run this sample locally, follow these instructions:
+
+In one terminal, [run Redis](https://redis.io/topics/quickstart):
```

Comment:
Is it better to refer to Redis Labs' [hosted redis solution](https://redislabs.com) here? (Maybe a question for the relevant PM.)

Reply:
I think the official Redis quickstart is better than that site for learning about what Redis is.